src/lib/unlang/xlat_builtin.c

```diff
@@ -3414,7 +3410,7 @@ do { \
 	XLAT_REGISTER_MONO("md5", xlat_func_md5, xlat_func_md5_arg);
 	xlat_register(NULL, "module", xlat_func_module, false);
 	XLAT_REGISTER_MONO("pack", xlat_func_pack, xlat_func_pack_arg);
-	xlat_register(NULL, "pairs", xlat_func_pairs, false);
+	XLAT_REGISTER_MONO("pairs", xlat_func_pairs, xlat_func_pairs_arg);
```

Comment:
This this actually an args xlat.  It's an xlat function with exactly one argument, which is the attribute reference.  The one attribute reference should be single valued.